pkg/tracing/otlp/otlp.go

```diff
@@ -86,3 +92,20 @@ func newTraceProvider(ctx context.Context, processor tracesdk.SpanProcessor, log
 	)
 	return tp
 }
+
+func buildSampler(samplerName string, samplerArg string) (tracesdk.Sampler, error) {
```

Comment:
Maybe we can follow same convention as jaeger here: https://github.com/thanos-io/thanos/blob/main/pkg/tracing/jaeger/config_yaml.go#L122
```suggestion
func getSampler(config Config) (tracesdk.Sampler, error) {
```

Reply:
Doing that in my next commit!